setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name="xgboost_ray",
     packages=find_packages(where=".", include="xgboost_ray*"),
-    version="0.1.14",
+    version="0.1.15",
```

Comment:
There is no need to change the version in this PR.

Reply:
Yeah, thanks for the comment. Will do!